codegra_fs/cgfs.py

```diff
@@ -1416,8 +1416,25 @@ def strippath(self, path: str) -> str:
     def load_courses(self) -> None:
         assert cgapi is not None
 
-        for course in cgapi.get_courses():
+        courses = cgapi.get_courses()
+        for dups in codegra_fs.utils.find_all_dups(
+            courses, lambda x: x['name']
+        ):
+            end = 4
+            if len(dups) != len(set(d['created_at'][:4] for d in dups)):
+                end = None
+
+            for dup in dups:
+                date = dup['created_at'][:end]
+                dup['name'] += ' - ' + date
+
+        for course in courses:
             assignments = course['assignments']
+            for dups in codegra_fs.utils.find_all_dups(
+                assignments, lambda x: x['name']
+            ):
+                for dup in dups:
+                    dup['name'] += ' - ' + dup['created_at']
```

Comment:
Same as above. Should we also do that for submissions in an assignment? Probably not remove the hours and minutes, maybe not even the seconds, but everything after that can be removed IMO.

Reply:
Agree, the seconds are also useless imo.